orchagent/muxorch.cpp

```diff
@@ -830,9 +836,90 @@ bool MuxOrch::isNeighborActive(IpAddress nbr, string alias)
         return ptr->isActive();
     }
 
+    string port;
+    if (!getMuxPort(mac, alias, port))
+    {
+        SWSS_LOG_INFO("Mux get port from FDB failed for '%s' mac '%s'",
+                       nbr.to_string().c_str(), mac.to_string().c_str());
+        return true;
+    }
+
+    if (!port.empty() && isMuxExists(port))
+    {
+        MuxCable* ptr = getMuxCable(port);
+        return ptr->isActive();
+    }
+
+    return true;
+}
+
+bool MuxOrch::getMuxPort(const MacAddress& mac, const string& alias, string& portName)
+{
+    portName = std::string();
+    Port rif, port;
+
+    if (!gPortsOrch->getPort(alias, rif))
+    {
+        SWSS_LOG_INFO("Interface '%s' not found in port table", alias.c_str());
```

Comment:
Q: If this is a true error case, then log level should be error and not info.
If this is expected to occur during normal operation perhaps due to race condition, then we should lower it to DEBUG level instead so that frequent encounter of this will not cause syslog to be filled up.

Reply:
This is not expected in our normal operation, so just added for a safety check. I'll change it to ERROR